rate/internal/crons/cron_scheduler.go

```diff
@@ -0,0 +1,51 @@
+package crons
+
+import (
+	"context"
+	"github.com/robfig/cron/v3"
+	"log"
+	"rate-service/internal/services"
+)
+
+const (
+	EveryHour = "0 * * * *"
```

Comment:
Do we really need to fetch currency rates every hour? It may produce a high amount of redundant data

Reply:
Well it depends on the frequency of the changes in currency rate, as for now it is changed once 24 hours due to NBU restrictions, but in case there are no restrictions there were cases when it was floating during the day. Summarizing it is not necessary to be that frequent nowadays and can be reduced to once 24 hours, so I will change it